docs/source/posts/2024/2024-08-21-final-report-robin.rst

```diff
@@ -0,0 +1,305 @@
+
+Contributions to FURY apart from the ones directly part of GSoC:
+    * https://github.com/fury-gl/fury/pull/862 - Rendering videos on a cube
+    * https://github.com/fury-gl/fury/pull/861 - docstring improvements
+    * https://github.com/fury-gl/fury/pull/891 - Codespell fix
+    * https://github.com/fury-gl/fury/pull/893 - .gitignore modification
+    * https://github.com/fury-gl/fury/issues/924 - Raised issue
+
+Contributions to other repositories during this time, due to GSoC work:
+    * https://github.com/langchain-ai/langchain/issues/23515 - Langchain issue raised
+    * https://github.com/github/docs/issues/34258 - GitHub issue raised
+    * https://github.com/github/docs/pull/34259 - PR for the raised GitHub issue
+    * https://github.com/orgs/community/discussions/136436 - GitHub feature request
+
+
+Acknowledgement
+---------------
+
+I am very thankful to my mentors `Serge Koudoro <https://github.com/skoudoro>`_ and `Mohamed Abouagour <https://github.com/m-agour>`_. They were awesome and provided me with a comfortable environment to work in. Also got to thank `Beleswar Prasad Padhi <https://www.linkedin.com/in/3v3ryone>`_ who gave me a very good introduction to opensource. The good thing about open source is I can still work on this (and other FURY projects) till I'm satisfied. I'm excited to continue contributing to the open source community.
+
+
+Timeline
+--------
+
+
+.. list-table:: GSoC 2024 Weekly Reports
+   :widths: 15 50 50
+   :header-rows: 1
+
+   * - Week
+     - Description
+     - Blog Post Link
+   * - Week 0
+     - Community Bonding!
+     - https://fury.gl/latest/posts/2024/2024-05-28-week-0-robin.html
+   * - Week 1
+     - It officially begins…
+     - https://fury.gl/latest/posts/2024/2024-06-06-week-1-robin.html
+   * - Week 2
+     - The first iteration!
+     - https://fury.gl/latest/posts/2024/2024-06-16-week2-robin.html
+   * - Week 3
+     - Data Data Data!
+     - https://fury.gl/latest/posts/2024/2024-06-16-week3-robin.html
+   * - Week 4
+     - Pipeline Improvements and Taking The Bot Public!
+     - https://fury.gl/latest/posts/2024/2024-07-01-week-4-robin.html
+   * - Week 5
+     - LLM Benchmarking & Architecture Modifications
+     - https://fury.gl/latest/posts/2024/2024-07-01-week-5-robin.html
+   * - Week 6
+     - UI Improvements and RAG performance evaluation
+     - https://fury.gl/latest/posts/2024/2024-07-27-week6-robin.html
+   * - Week 7
+     - Surviving final examinations
+     - https://fury.gl/latest/posts/2024/2024-07-27-week7-robin.html
+   * - Week 8
+     - Gemini Finetuning
+     - https://fury.gl/latest/posts/2024/2024-07-27-week8-robin.html
+   * - Week 9
+     - None
+     - None
+   * - Week 10
+     - None
+     - None
+   * - Week 11
+     - None
+     - None
+   * - Week 12
+     - None
+     - None
```

Comment:
merged, you can update this

Reply:
@skoudoro, I can't find the links to the blog. https://fury.gl/latest/blog.html